### PR TITLE
x86: replace x86_64 crate with simple outl wrapper

### PR DESCRIPTION
**Pull request by Skallwar**
We use the x86_64 crate only to make a outl. Imao this is overkill

### CODE REVIEW

File: src/x86.rs [hunk 1]
```diff
@@ -18,6 +18,15 @@ pub struct X86 {
     custom_exit_success: u32,
 }
 
+/// Output a long on an io port
+pub fn outl(io_base: u16, code: u32) {
```
Comment (andre-richter) [inline src/x86.rs:22]:
```suggestion
fn outl(io_base: u16, code: u32) {
```

File: src/x86.rs [hunk 2]
```diff
@@ -32,15 +41,12 @@ impl X86 {
...
-
-        let mut port = Port::<u32>::new(self.io_base);
-        unsafe { port.write(code.into()) }; // QEMU will execute `exit(((code << 1) | 1))`.
+        outl(self.io_base, code); // QEMU will execute `exit(((code << 1) | 1))`.
 
-        // For the case that the QEMU exit attempt did not work, transition into an infinite loop.
-        // Calling `panic!()` here is unfeasible, since there is a good chance this function here is
-        // the last expression in the `panic!()` handler itself. This prevents a possible infinite
-        // loop.
+        // For the case that the QEMU exit attempt did not work, transition into an infinite
```
Comment (andre-richter) [inline src/x86.rs:46]: please don't change the formatting.

File: src/x86.rs [hunk 3]
```diff
@@ -18,6 +18,15 @@ pub struct X86 {
     custom_exit_success: u32,
 }
 
+/// Output a long on an io port
+pub fn outl(io_base: u16, code: u32) {
+    unsafe {
+        llvm_asm!("outl %eax, %dx" :: "{dx}"(io_base), "{eax}"(code) :: "volatile");
+        // asm!("outl dx, eax", in("dx")io_base, in("eax")code); I don't know why this is not
```
Comment (andre-richter) [inline src/x86.rs:25]: Would be nice if we can get it working with the new asm! syntax.
I'll try to switch the aarch64 version over to it.

If both was successful, I would release the new crate version.

File: src/x86.rs (hunk 2 above)
Comment (Skallwar) [reply src/x86.rs]: Sorry, this must be a typo. Fixed e521c2d

File: src/x86.rs (hunk 3 above)
Comment (Skallwar) [reply src/x86.rs]: For some reason, new asm! syntax does not accept outl but out is working

**Pull request merged**